app/templates/courses/courses-view.jade

```diff
@@ -37,6 +37,44 @@ block content
 
           else
 
+            if view.showTournament || view.awaitingTournament
+              .tournament-logo
+                img(src="/images/pages/courses/tarena-discovery-cup-tournament-logo.png")
+              .text-center
+                h1#tournament-title(data-i18n="ladder.live_tournament")
+                a#tournament-rules(data-i18n="ladder.tournament_rules")
+                #tournament-rules-content.hide.rtl-allowed
+                  if /^zh/.test(me.get('preferredLanguage', true))
+                    p Welcome to the Tarena CodeCombat live tournament! Compete for the top 200 spots to qualify for the next round.
+
+                    p Scoring is as follows: your code will play constantly in the background against other players to determine how strong it is. The number of points you earn for a match are determined by a Bayesian calculation predicting how likely you were to win that match. So if you beat a player ranked more highly than you, you gain more points. Beat a player lower than you, gain fewer points. Lose to a weaker player, lose more points. Lose to a stronger player, lose fewer points. Your code will play on both the red and blue teams, but scoring will be based on the red team.
+
+                    p You can test your code against any player and improve your strategy until you win, but remember: no points are scored until you submit your latest code and it starts to automatically plays matches against other players.
+                  else
```

Comment:
Is there any difference between the text in the above condition (lines 48 - 52), vs below at line 54-58? They look the same - do we need if-else here?

Reply:
Right, this is just a placeholder for now; when I get Chinese translations from the China team, I'll put them in here.

Reply:
Should this be moved to en.coffee then?